lib/dice.py

```diff
@@ -0,0 +1,24 @@
+from random import randint
+import re
+
+class Dice():
+
+    def split():
+        numbers = re.split('[d+]', roll)
+        return numbers
+
+    @static def roll(self, numbers):
```

Comment:
`@static` should go on the line above.

Reply:
moved static

Reply:
hmm, it doesn't like that static is there, saying NameError: name 'static' is not defined